sys-libs/glibc/glibc-9999.ebuild

```diff
@@ -698,21 +692,29 @@ sanity_prechecks() {
 			die "No __thread support in gcc!"
 		fi
 
-		if ! is_crosscompile && ! tc-is-cross-compiler ; then
-			# Building fails on an non-supporting kernel
-			ebegin "Checking running kernel version (${run_kv} >= ${want_kv})"
-			if ! eend_KV ${run_kv} ${want_kv} ; then
-				echo
-				eerror "You need a kernel of at least ${want_kv}!"
-				die "Kernel version too low!"
+		if [[ ${KERNEL} == linux ]] ; then
```

Comment:
I believe the correct way of checking for kernel type is to `use kernel_linux`. But it works only for non-cross case.

The problem is that `crossdev` builds glibc on current host's profile (and uses host's kernel setting), and `cross-emerge` builds it on target's profile. See table for all supported cases: https://gitweb.gentoo.org/repo/gentoo.git/tree/sys-libs/glibc/glibc-2.27-r5.ebuild#n40

Thus I suggest using the guard `if [[ ${CTARGET} == *-linux* ]]; ` for this check to depend less on profile sanity.